src/main/java/bio/terra/service/tabulardata/google/BigQueryPdao.java

```diff
@@ -129,6 +130,7 @@ public void createDataset(Dataset dataset) throws InterruptedException {
                     datasetName, table.getSoftDeleteTableName(), buildSoftDeletesSchema());
                 bigQuery.create(buildLiveView(bigQueryProject.getProjectId(), datasetName, table));
             }
+            // todo: don't catch generic exceptions
```

Comment:
Do you have a sense for what the explicit exceptions might be?

Reply:
IMO it's better to make a tech debt ticket than add a TODO, since we don't have a policy for scheduling TODO work. (Although if you do add a TODO, please spell it `TODO` so the intellij scanner will see it.)

Reply:
I'll make a ticket, thanks!